src/python/pants/option/global_options.py

```diff
@@ -475,860 +486,764 @@ def from_options(cls, options: OptionValueContainer) -> LocalStoreOptions:
-                "store on disk) it also bounds the per-file size to (VALUE / "
-                f"`{local_store_shard_count_flag}`)."
-                "\n\n"
-                "This value doesn't reflect space allocated on disk, or RAM allocated (it "
-                "may be reflected in VIRT but not RSS). However, the default is lower than you "
-                "might otherwise choose because macOS creates core dumps that include MMAP'd "
-                "pages, and setting this too high might cause core dumps to use an unreasonable "
-                "amount of disk if they are enabled."
-            ),
-            default=DEFAULT_LOCAL_STORE_OPTIONS.files_max_size_bytes,
-        )
-        register(
-            "--local-store-directories-max-size-bytes",
-            type=int,
-            advanced=True,
-            help=(
-                "The maximum size in bytes of the local store containing directories. "
-                f"Stored below `{local_store_dir_flag}`."
-            ),
-            default=DEFAULT_LOCAL_STORE_OPTIONS.directories_max_size_bytes,
-        )
-        register(
-            "--named-caches-dir",
-            advanced=True,
-            help=(
-                "Directory to use for named global caches for tools and processes with trusted, "
-                f"concurrency-safe caches.\n\n{cache_instructions}"
-            ),
-            default=os.path.join(get_pants_cachedir(), "named_caches"),
-        )
+    Note that regular code can also access these options as normal global-scope options. Their
+    status as "bootstrap options" is only pertinent during option registration.
+    """
 
-        register(
-            "--local-execution-root-dir",
-            advanced=True,
-            help=(
-                f"Directory to use for local process execution sandboxing.\n\n{cache_instructions}"
-            ),
-            default=tempfile.gettempdir(),
-        )
-        register(
-            "--local-cache",
-            type=bool,
-            default=DEFAULT_EXECUTION_OPTIONS.local_cache,
-            help=(
-                "Whether to cache process executions in a local cache persisted to disk at "
-                "`--local-store-dir`."
-            ),
-        )
-        register(
-            "--process-cleanup",
-            type=bool,
-            default=DEFAULT_EXECUTION_OPTIONS.process_cleanup,
-            help=(
-                "If false, Pants will not clean up local directories used as chroots for running "
-                "processes. Pants will log their location so that you can inspect the chroot, and "
-                "run the `__run.sh` script to recreate the process using the same argv and "
-                "environment variables used by Pants. This option is useful for debugging."
-            ),
-        )
+    buildroot = get_buildroot()
+    default_distdir_name = "dist"
+    default_rel_distdir = f"/{default_distdir_name}/"
```

Comment:
Should probably be private, given that now we have each option as a property whereas before you had to use `options.bootstrap`